### PR TITLE
Add documentation for the scripts defined in the package.json

### PR DESCRIPTION
**Pull request by robpal94**
## What did you change?

This PR adds a documentation for the predefined scripts in the _package.json_ file which are callable via `npm run script`. 
The idea was to give a kind of overview of maybe helpful scripts. 

## How can others test the changes?

* Look at the readme

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [ ] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [ ] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [ ] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,6 +161,22 @@ Die Tests starten:
 npm run integration-test
 ```
 
+## Shortcut scripts
+
+Es sind bereits einige Script in der `package.json` - Datei vordefiniert, welche
```
Comment (alexanderkasten) [inline README.md]: hier schreibst du Script

File: README.md [hunk 2]
```diff
@@ -161,6 +161,22 @@ Die Tests starten:
...
+sich einfach durch `npm run <name>` ausführen lassen.
+
+Die folgenden vordefinierten Skripte könnten hilfreich sein:
```
Comment (alexanderkasten) [inline README.md]: und hier Skripte

File: README.md [hunk 3]
```diff
@@ -161,6 +161,22 @@ Die Tests starten:
...
+* `build`: Baut das Aurelia Bundle
+* `start`: Startet die BPMN - Studio Webanwendung
```
Comment (alexanderkasten) [inline README.md]: BPMN-Studio

File: README.md [hunk 4]
```diff
@@ -161,6 +161,22 @@ Die Tests starten:
...
+* `start_dev`: Startet die BPMN - Studio Webanwendung und trackt die Quelldatein (geänderte Quelltextdatein werden neu transpiliert und die Webanwendung wird neu geladen)
```
Comment (alexanderkasten) [inline README.md]: BPMN-Studio

File: README.md [hunk 5]
```diff
@@ -161,6 +161,22 @@ Die Tests starten:
...
+* `electron-start-dev`: Baut das Aurelia bundle und startet die Electron Anwendung
```
Comment (alexanderkasten) [inline README.md]: Bundle

File: README.md [hunk 6]
```diff
@@ -161,6 +161,22 @@ Die Tests starten:
...
+Die folgenden vordefinierten Skripte könnten hilfreich sein:
+* `build`: Baut das Aurelia Bundle
+* `start`: Startet die BPMN-Studio Webanwendung
+* `start_dev`: Startet die BPMN-Studio Webanwendung und trackt die Quelldatein (geänderte Quelltextdatein werden neu transpiliert und die Webanwendung wird neu geladen)
+* `electron-start-dev`: Baut das Aurelia Bundle und startet die Electron Anwendung
+* `reset`: Entfernt alle node_modules, die `package-lock.json` Datei und bereinigt den NPM - Cache
+* `lint`: Startet `tslint` für das gesamte Projekt
+* `electron-build-macos`: Baut die Electron - Anwendung für MacOS
+* `electron-build-linux`: Baut die Electron - Anwendung für Linux
+* `electron-build-windows`: Baut die Electron - Anwendung für Windows
```
Comment (cmg-dev) [inline README.md]: @robpal94 lass und morgen da drüber sprechen.

ich habe ein paar Vorschläge, was wir hier besser machen können.

**Comment by cmg-dev** [comment]
Review mit @robpal94 zusammen durchgeführt. 👌

**Pull request merged**